ml-agents/mlagents/trainers/trainer/rl_trainer.py

```diff
@@ -235,6 +235,7 @@ def advance(self) -> None:
                 with hierarchical_timer("_update_policy"):
                     if self._update_policy():
                         for q in self.policy_queues:
+                            time.sleep(10)
```

Comment:
was this intentional?

Reply:
Sorry, didn't mean to check that in. Reverting it.